src/mol-model-formats/structure/property/bonds/struct_conn.ts

```diff
@@ -21,6 +21,8 @@ import { FormatPropertyProvider } from '../../common/property';
 export interface StructConn {
     readonly data: Table<mmCIF_Schema['struct_conn']>
     readonly byAtomIndex: Map<ElementIndex, ReadonlyArray<StructConn.Entry>>
+    // Cantor pairs of residue indices that have a struct-conn record
```

Comment:
so it shows up in editor tooltips...
```suggestion
    /** Cantor pairs of residue indices that have a struct-conn record */
```